rules/falco_rules.yaml

```diff
@@ -2687,8 +2689,13 @@
 # Whitelist for known docker client binaries run inside container
 # - k8s.gcr.io/fluentd-gcp-scaler in GCP/GKE
 - macro: user_known_k8s_client_container
-  condition: (k8s.ns.name="kube-system" and container.image.repository=k8s.gcr.io/fluentd-gcp-scaler)
- 
+  condition: >
+    (k8s.ns.name="kube-system" and container.image.repository=k8s.gcr.io/fluentd-gcp-scaler) or
+    container.image=mcr.microsoft.com/aks/hcp/hcp-tunnel-front
+
+- macro: user_known_k8s_client_container_parens
```

Comment:
is this macro used by any rule?

Reply:
Oh sorry, I was editing things and missed this. I wanted to replace the use of user_known_k8s_client_container in the rule's condition. Fixed.